File: ex01/Java/Ex01.java
import java.util.*;

class Ex01 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("What is your name? ");
        String myName = scanner.next();

        scanner.close();

        // String format 
        System.out.println(String.format("Hello, %s, nice to meet you!", myName));
        // System.out.printf("Hello, %s, nice to meet you!", myName);
        // System.out.println("Hello, " + myName + ", nice to meet you!");
    }
}

File: ex01/C/ex01.c
/**
 * ex01. 인사하기
 */
#include <stdio.h>

int main(int argc, char* argv[])
{
    char name[20] = "";

    printf("What is your name? " );

    scanf("%s", name);

    printf("Hello, %s, nice to meet you!", name);

    return 0;
}